scenarios/regex_random_pod_kill.yml

```diff
@@ -0,0 +1,30 @@
+config:
+  # we don't set loopsNumber, so it will loop indefinitely
+  minSecondsBetweenRuns: 1
+  maxSecondsBetweenRuns: 30
+
+nodeScenarios: []
+podScenarios:
+  - name: "delete random openshift, running pods from  all namespaces"
+
+    match:
+      - namespace:
+          name: "*"
+
+    filters:
+
+      # property filters (all the property filters support regexp)
+      - property:
+          name: "name"
+          value: "openshift-*"
```

Comment:
I might be wrong but think this will fetch pods in all the namespaces and kill the pods starting with openshift-* ( will miss pods like apiserver, etcd e.t.c since they don't have an openshift prefix ) vs fetching pods in openshift-* namespaces and killing them. Thoughts?

Reply:
I think you are correct about the current implementation only getting the pods from all namespaces that have "openshift-" in the name. I'm not sure which is the better implementation, I think the current implementation is the one described in the original issue but not sure if pods in openshift-* namespaces would be better. @mffiedler thoughts?

Reply:
Think the filter is being applied to the pods meaning it will kill the pods whose name starts with openshift-* ( we will miss apiserver, etcd and other pods since they don't have a openshift prefix in their name ). We might want to place the filters for the namespaces meaning it should the pods running in the namespaces matched by regex i,e openshift-* in this case. The current config maybe doing the same and there's a high possibility that I might have missed it -:)